src/httpcodes.rs

```diff
@@ -5,7 +5,7 @@ use httpresponse::{HttpResponse, HttpResponseBuilder};
 
 macro_rules! STATIC_RESP {
     ($name:ident, $status:expr) => {
-        #[allow(non_snake_case)]
+        #[allow(non_snake_case, missing_docs)]
```

Comment:
macros are evil but you could play with transforming the `$name:ident` to a `$name:expr` and putting it in a `#[doc = $name]` attr